pkg/webhooks/techpreviewnoupgrade/techpreviewnoupgrade.go

```diff
@@ -131,8 +132,15 @@ func hasTechPreviewNoUpgrade(features []string) bool {
 
 func NewWebhook() *TechPreviewNoUpgradeWebhook {
 	scheme := runtime.NewScheme()
-	admissionv1.AddToScheme(scheme)
-	corev1.AddToScheme(scheme)
+
+	err := admissionv1.AddToScheme(scheme)
+	if err != nil {
+		logx.Fatal(err)
```

Comment:
Why `logx` over the existing `logf` package that's already imported?

Reply:
done